docs/iis-instrumentation.md

```diff
@@ -62,6 +70,20 @@ to set it for all ASP.NET application running in Integrated Pipeline Mode:
   </location>
 ```
 
+> **Note** After applying above changes you can check whether `opentelemetry modules`
+> are loaded by using `appcmd` command.
+> Following example shows invocation for `WebDemo\` application:
+
+```terminal
+  appcmd list modules /app.name:"WebDemo/"
```

Comment:
Maybe make it clear that this appcmd is found under %SystemDrive%\Windows\system32\inetsrv ?

Reply:
Sorry, I merged before this could be addressed.